read-gtfs/mapping.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const randomId = require('unique-string')
+
+const dataToDB = {
+	agency: a => 'agency-' + a.agency_id,
+	stops: s => 'stop-' + s.stop_id,
+	routes: r => 'route-' + r.route_id,
+	trips: t => 'trip-' + t.route_id + '-' + t.trip_id,
+	stop_times: st => 'stop_time-' + randomId() + '-' + st.trip_id,
+	calendar: s => 'service-' + s.service_id,
+	calendar_dates: e => 'calendar_date-' + randomId() + '-' + e.service_id
+}
+
+const dbToData = {
+	agency: agencyId => 'agency-' + agencyId,
+	stops: stopId => 'stop-' + stopId,
+	routes: routeId => 'route-' + routeId,
+	trips: (routeId, tripId) => 'trip-' + routeId + '-' + tripId,
```

Comment:
This won't work if `routeID` is empty.

Reply:
When will `routeId` be empty?

Reply:
When all trips are requested. In the legacy `tripStream` method for example.

Reply:
Ah. I changed this mapping to work with ranges of e.g. trips in my upcoming PR.

Let me know if you want these lines out of this PR, then I will move them to the other one.

Reply:
I'd prefer to either have all changes in one PR or two strictly separated ones.

Reply:
Therefore yes, or add some commits to this PR, I don't care 😄